backend/python/app/graphql/types/story_type.py

```diff
@@ -33,3 +33,15 @@ class StoryRequestDTO(graphene.InputObjectType):
     youtube_link = graphene.String(required=True)
     level = graphene.Int(required=True)
     translated_languages = graphene.List(LanguageEnum)
+
+
+class StoryTranslationResponseDTO(graphene.ObjectType):
+    story_id = graphene.Int(required=True)
+    title = graphene.String(required=True)
+    description = graphene.String(required=True)
+    level = graphene.Int(required=True)
+    story_translation_id = graphene.Int(required=True)
+    language = graphene.String(required=True)
+    stage = graphene.Field(StageEnum, required=True)
+    translator_id = graphene.Int(required=True)
```

Comment:
`translator_id ` may be null if a translator dropped the story (rare edgecase, we will have to build a way for the translation to be picked up again) 
`reviewer_id ` will be null if the story is in START or TRANSLATION stage 
TLDR these fields are not required